app/src/main/res/menu/countries_menu.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/discover_hide_item"
+        android:checkable="true"
+        android:enabled="true"
+        android:title="@string/discover_hide"
+        android:visible="true" />
```

Comment:
Please set showAsAction to never, so that it is always displayed in the overflow menu

Reply:
fixed